libcontainer/cgroups/fs/cpuset_test.go

```diff
@@ -65,3 +203,64 @@ func TestCpusetSetMems(t *testing.T) {
 		t.Fatal("Got the wrong value, set cpuset.mems failed.")
 	}
 }
+
+func TestCpusetStatsCorrect(t *testing.T) {
+	helper := NewCgroupTestUtil("cpuset", t)
+	defer helper.cleanup()
+	helper.writeFileContents(cpusetTestFiles)
+
+	cpuset := &CpusetGroup{}
+	actualStats := *cgroups.NewStats()
+	err := cpuset.GetStats(helper.CgroupPath, &actualStats)
+	if err != nil {
+		t.Fatal(err)
+	}
+	expectedStats := cgroups.CpusetStats{
+		Cpus:                  []uint16{0, 1, 2, 7, 12, 13, 14},
+		CpuExclusive:          1,
+		Mems:                  []uint16{1, 2, 3, 4, 6, 9},
+		MemoryMigrate:         1,
+		MemHardwall:           0,
+		MemExclusive:          0,
+		MemorySpreadPage:      0,
+		MemorySpreadSlab:      1,
+		MemoryPressure:        34377,
+		SchedLoadBalance:      1,
+		SchedRelaxDomainLevel: -1}
+	if !reflect.DeepEqual(expectedStats, actualStats.CpusetStats) {
+		t.Errorf("Expected Cpuset stats usage %#v but found %#v\n",
+			expectedStats, actualStats.CpusetStats)
+	}
+
+}
+
+func TestCpusetStatsFails(t *testing.T) {
+	var errStrings []string
```

Comment:
you can avoid piling up errors if you use `t.Errorf()` instead of `t.Fatalf()`.

Reply:
@piowag ^^^

Reply:
Applied